tools/src/main/scala/org/apache/spark/sql/rapids/tool/AppFilterImpl.scala

```diff
@@ -76,7 +77,18 @@ class AppFilterImpl(
         if (appNameOpt.isDefined) {
           appNameOpt.get.equals(filterAppName)
         } else {
-          // in complete log file
+          // incomplete log file
+          false
+        }
+      }
+      filtered.map(_.eventlog).toSeq
+    } else if (appArgs.startAppTime.isSupplied) {
```

Comment:
Should we support both the cases? i.e If applciationName is given along with time range in the commandline. 
 --application-name "appName" --start-app-time "1w"
If AND is not supported, then we can document that. I will follow the same in my follow on PR's for limit.

Reply:
yeah that is a good point we talked about this, it is officially a P1 so perhaps lets do taht as a follow on once we have all the filtering features in and make sure we have tests that will cover all the cases.